internal/cli/atlas/customdbroles/create.go

```diff
@@ -28,7 +28,7 @@ import (
 	atlas "go.mongodb.org/atlas/mongodbatlas"
 )
 
-const createTemplate = "Custom Database Role {{.RoleName}} successfully created.\n"
+const createTemplate = "Custom database role '{{.RoleName}}' successfully created.\n"
```

Comment:
[q] Why can't we provide this functionality to the create command?

Reply:
what do you mean here?

Reply:
I am just wondering why we are not allowing the user to create a custom role with an action which is applied to multiple resources with the `create `command

something like :
`./bin/mongocli atlas customDBRoles create myrole5 --privilege FIND@db1.coll2,FIND@db1.ss,UPDATE@db2.coll3
Error: POST https://cloud-dev.mongodb.com/api/atlas/v1.0/groups/5e4e593f70dfbf1010295836/customDBRoles/roles: 400 (request "ATLAS_DUPLICATE_CUSTOM_ROLE_ACTION") A custom role cannot have duplicate actions.`

Reply:
but I see now that the `joinActions` should allow this

Reply:
Oh I thought for this comment it was about the line and I was confused